src/barChart.ts

```diff
@@ -260,9 +325,12 @@ module powerbi.extensibility.visual {
                     height: d => height - yScale(<number>d.value),
                     y: d => yScale(<number>d.value),
                     x: d => xScale(d.category),
-                    fill: d => d.color,
                 })
-                .style('fill-opacity', viewModel.settings.generalView.opacity / 100);
+                .style({
+                    'fill-opacity': viewModel.settings.generalView.opacity / 100,
```

Comment:
Please add stroke opacity for highlight indication in high-contrast mode (native use `stroke-opacity 0.4`)

Reply:
done. Thanks for your notes.